\--- file1.txt 2025-04-21 23:14:00.000000000 -0700 +++ file2.txt 2025-04-21 23:14:00.000000000 -0700 @@ -1,5 +1,5 @@ Line 1 -Line 2 +Line 2 modified Line 3 Line 4 -Line 5 +Line 5 deleted

@@ -6,5 +6,5 @@ Line 6 Line 7 -Line 8 +Line 8 modified Line 9 Line 10